src/apollo/getAverageBlockTime.ts

```diff
@@ -1,83 +1,84 @@
-import { blockClient, blockClient_matic } from './client'
-import { blockQuery, blocksQuery } from './queries'
-import { getUnixTime, startOfHour, startOfMinute, startOfSecond, subDays, subHours } from 'date-fns'
+export {}
+// import { blockClient, blockClient_matic } from './client'
```

Comment:
why is this file commented out? if not required remove it.

Reply:
leftover indeed, done